### ISSUE TITLE
Color contrast ratio of keyboard focus present on "C#" control is 1.277:1 which is less than 3:1: A11y_Azure Function_Create new project_Non-text Contrast

### ISSUE THREAD

**Issue by Rama8345** [microsoft/vscode-azurefunctions#4331]
### GIT Hub Tags:
#A11yTCS; #A11ySev2; #Azure Tools for VS Code; #BM_Azure Tools for VS Code_Win32_Nov2024; #DesktopApp; #A11yMAS; #WCAG1.4.11; #Non-text Contrast; #AILimited; #Win11; #FTP;

### Environment and OS details:
Application name: Azure Tools for VS Code
URL: Visual studio code

### Reproduction Steps:

1. Open Visual studio code 
2. TAB till extension button and press ENTER then search for "Azure Functions"
3. TAB till install and press ENTER. 
4. TAB till "Azure" button and press ENTER.
5. TAB till "Azure function" control and hit ENTER.
6. TAB till "Create new project" control and hit ENTER.
7. Open accessibility insights for windows and check color contrast ratio of keyboard focus on "C#".
8. Observe the issue whether Color contrast ratio of keyboard focus present on "C#" control is less than 3:1 or not.

### Actual behavior:
Color contrast ratio of keyboard focus present on "C#" control is 1.277:1 which is less than 3:1

### Expected behavior:
Color contrast ratio of keyboard focus present on "C#" control should be greater than or equals to 3:1

### User Impact:
Low vision users will not be able to understand where the keyboard focus is present if color contrast ratio will be less than 3:1.

### Attachment:
![Image](https://github.com/user-attachments/assets/318bf0aa-cfd7-4d20-90dc-555e199d4ae0)

**Comment by PoojaNamde** [comment]
#Rev:PoojaNamde;

**Comment by MicroFish91** [comment]
@Rama8345 Could you let me know the name of the VS Code color theme you are using here?

F1 > Preferences: Color Theme

**Comment by PoojaNamde** [comment]
@MicroFish91 Default dark theme we used to log this bug.

**Comment by MicroFish91** [comment]
Tracking upstream issue on VS Code repo https://github.com/microsoft/vscode/issues/237198

**Comment by PoojaNamde** [comment]
@MicroFish91 we have logged new bug https://github.com/microsoft/vscode-azurefunctions/issues/4482 to track this issue. Could you please close this one.
